src/content/guide/tools-and-features/remote-diagnostics.md

```diff
@@ -0,0 +1,320 @@
+device falling into an unhealthy state, your team will be empowered with
+rich context and suggested courses of action to quickly diagnose and
+rapidly resolve the issue.
+
+{{#if electron}}
+<img
+src="/assets/images/remote-diagnostics/successful-diagnostics-test.png"
+class="full-width"/>
+{{else}}
+<img
+src="/assets/images/remote-diagnostics/successful-diagnostics-test-wifi.png"
+class="full-width"/>
+{{/if}}
+<p class="caption">Remote Diagnostics allow you to test the connectivity
+health of your devices, and quickly resolve problems when they arise.</p>
+
+## Connectivity Layers
+
+Multiple connectivity layers must be operating successfully for a given device
+to be able to successfully communicate with the cloud. Note that the
+relevant connectivity layers vary based on the type of device (i.e.
+Wi-Fi vs. Cellular).
+
+These connectivity layers are:
+
+{{#if electron}}
+  <img class="full-width" alt="Device, SIM Card, Cellular Network, and
+  Particle Cloud"
+  src="/assets/images/remote-diagnostics/connectivity-layers-cellular.png"/>
+{{else}}
+  <img alt="Device and
+  Particle Cloud"
+  src="/assets/images/remote-diagnostics/connectivity-layers-wifi.png"/>
+{{/if}}
+
+
+### Device
+The device itself must be in a healthy state in order to successfully
+communicate with the cloud. A variety of factors influence its state,
+such as battery state of charge, signal strength, available memory, and
+application firmware that does not exceed enforced rate limits.
+
+{{#if electron}}
+### SIM Card
+Cellular devices rely on a SIM card to facilitate a connection to the
+cellular network. This SIM must be in an active state, allowing the
+device to try to initiate a data session with the network.
+
+### Cellular Network
+
+In addition to the need for an active SIM, the device still must be in
+range of a cell tower to create a healthy connection to the cellular
+network. Particle works with a global network of cellular carriers to
+allow devices to connect virtually anywhere in the world.
+{{/if}}
+
+### Particle Cloud
+
+The health of the Particle Cloud is critical to devices having the ability to
+successfully connect and communicate. There are a few Particle Cloud
+services in particular that directly impact device health and
+communications:
+
+#### Device Service
+
+The Device Service brokers the connection between an IoT device and
+the Particle Cloud. In addition, the Device Service is responsible for
+shuttling of messages to and from a Particle device.
+
+#### API
+
+The API provides a REST interface to allow remote interactions with
+Particle devices and the cloud. This includes calling a [
+function](/reference/api/#call-a-function),
+checking a [variable](/reference/api/#get-a-variable-value),
+or publishing an [event](/reference/api/#publish-an-event)
+that devices subscribe to.
+
+#### Webhooks
+
+The Webhooks service allows for device data to be sent to other
+apps and services. Webhooks also allow for devices to ingest
```

Comment:
"also allow for devices" => "also allows devices"

Reply:
good call